internal/spec/build.go

```diff
@@ -69,8 +71,9 @@ func FromEvaluation(ctx context.Context, ses []interpreter.SideEffect, now time.
 
 	if len(spec.Operations) == 0 {
 		return nil,
-			fmt.Errorf("this Flux script returns no streaming data. " +
-				"Consider adding a \"yield\" or invoking streaming functions directly, without performing an assignment")
+			errors.New(codes.FailedPrecondition,
```

Comment:
Can you update this to `codes.Invalid`? I like to use `codes.Invalid` for when the problem is data-independent and `codes.FailedPrecondition` when it's data dependent.